src/main/java/hudson/os/windows/ManagedWindowsServiceLauncher.java

```diff
@@ -181,10 +181,10 @@ private AccountInfo getLogOn() {
     }
 
     @Override
-    public void launch(final SlaveComputer computer, final TaskListener listener) throws IOException, InterruptedException {
+    public void launch(SlaveComputer computer, TaskListener listener) throws IOException, InterruptedException {
```

Comment:
why?

Reply:
Have a look at the commit 2a17de4903cebd885ef76a85d7633665677bbb52 where the `final` modifiers have been removed. For me it looked like they have only been thrown in arbitrarily. Therefore I remove these and left the ones on class field as they are more common, have a bigger scope and impact.